### PR TITLE
cleanup cloud migration code

### PR DESCRIPTION
**Pull request by trevj**
Just a little random cleanup...this was bothering me.

### CODE REVIEW

File: testing/run-scripts/run_cloud.sh [hunk 1]
```diff
@@ -53,97 +53,74 @@ while getopts p:z:s:i:uwd:b:ah? opt; do
...
-  fi
+  echo "-u must be used when -w is used"
+  usage
 fi
 
-# Migrate the giver and getter accounts' authorized_keys.
-if [ "$WIPE" = false ]
+# Set banner and hostname/IP if none were specified on the command line or
+# migrated from the previous installation.
+if [ -z "$BANNER" ]
```
Comment (soycode) [inline testing/run-scripts/run_cloud.sh]: This assumes DO, which is fine for now but I'm wondering if it's worth setting and checking some sort of $DO variable for futureproofness (which would be default true for now, but potentially override-able with flags).

File: testing/run-scripts/run_cloud.sh [hunk 2]
```diff
@@ -53,97 +53,74 @@ while getopts p:z:s:i:uwd:b:ah? opt; do
...
-if [ "$UPDATE" = true ]
+if [ -z "$PUBLIC_IP" ]
 then
-  docker rm -f uproxy-sshd || true
-  docker rm -f uproxy-zork || true
-  docker rmi $SSHD_IMAGE || true
-  # TODO: This will fail if there are any containers using the
-  #       image, e.g. run_pair.sh. Regular cloud users won't be.
-  docker rmi $ZORK_IMAGE || true
+  # Beautiful cross-platform one-liner cogged from:
```
Comment (soycode) [inline testing/run-scripts/run_cloud.sh]: Incredibly not important, but I think maybe s/cogged/copped/.

File: testing/run-scripts/run_cloud.sh (hunk 1 above)
Comment (trevj) [inline testing/run-scripts/run_cloud.sh]: How does it assume DO? The `curl` command times out after 2s.

File: testing/run-scripts/run_cloud.sh (hunk 2 above)
Comment (trevj) [inline testing/run-scripts/run_cloud.sh]: "copped"? :-)

Comment (soycode) [inline testing/run-scripts/run_cloud.sh]: Heh I think it's a closer use of some form of slang/idiom - that said, "grabbed" is also appropriate.

File: testing/run-scripts/run_cloud.sh (hunk 1 above)
Comment (soycode) [inline testing/run-scripts/run_cloud.sh]: Ah k, and I'm assuming it doesn't return anything of DO doesn't own the IP. If so, then makes sense.

**Comment by soycode** [comment]
:+1:

**Pull request merged**